cognite/client/_api/time_series.py

```diff
@@ -10,7 +10,12 @@ class TimeSeriesAPI(APIClient):
     _LIST_CLASS = TimeSeriesList
 
     def __call__(
-        self, chunk_size: int = None, include_metadata: bool = False, asset_ids: List[int] = None, limit: int = None
+        self,
+        chunk_size: int = None,
+        include_metadata: bool = False,
+        asset_ids: Optional[List[int]] = None,
+        root_asset_ids: Optional[List[int]] = None,
```

Comment:
Should we call this root_ids instead of root_asset_ids, for consistency with other .list() methods?

Reply:
still not sure of this. current suggestion is to match the API:
assets:  parent_ids, root_ids (=dict)
time_series: asset_ids, root_asset_ids (=list)

You would prefer to match root_ids parameter names in assets rather than the 'asset_ids' name in the other time_series parameter?
assets:  parent_ids, root_ids (=dict)
time_series: asset_ids, root_ids (=list)